public/content/it/welcome/welcome-to-d.md

```diff
@@ -0,0 +1,58 @@
+# Benvenuto al tour di D
+
+Benvenuto al tour interattivo del *Linguaggio di Programmazione D*.
+
+{{#dmanmobile}}
+
+Questo tour ti fornirà le basi di questo  __potente__ ed __espressivo__
+linguaggio, che compila in __efficiente__ codice __nativo__.
+
+{{/dmanmobile}}
+
+### Cos'è D?
+
+D rappresenta il culmine di  _decenni di esperienza nell'implementazione di compilatori_
+per molti linguaggi differenti e dispone di un gran numero di
+[feature uniche](http://dlang.org/overview.html):
+
+{{#dmandesktop}}
+
+- costrutti _ad alto livello_ per una grande capacità di modellazione
+- linguaggio compilato in codice _ad alte prestazioni_
+- tipizzazione statica
+- evoluzione del linguaggio C++ (senza ripeterne gli errori)
+- interfacciamento diretto con le API del sistema operativo e con l'hardware
+- tempi di compilazione estremamente rapidi
+- programmazione memory-safe (SafeD)
+- codice _mantenibile_ e _facile da comprendere_
+- bassa curva di apprendimento (sintassi C-like, simile a Java e molti altri linguaggi)
+- compatibile con il codice C a livello di ABI
+- linguaggio multi-paradigma (programmazione imperativa, strutturata, orientata agli oggetti, generica, funzionale e persino codice assembly integrato)
+- costrutti di prevenzione degli errori (asserzioni, contratti, unit testing)
+
+... e molte altre [features](http://dlang.org/overview.html).
```

Comment:
"Caratteristiche" is quite a neutral term (in the previous spot, it was made positive by "unique"), while "features" is inherently positive. So what about a stronger word like "peculiarità" or "qualità"?